contrib/registry/eureka/client.go

```diff
@@ -0,0 +1,363 @@
+package eureka
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"strings"
+	"sync"
+	"text/template"
+	"time"
+)
+
+const (
+	statusUp            = "UP"
+	statusDown          = "DOWN"
+	statusOutOfServeice = "OUT_OF_SERVICE"
+	heartbeatRetry      = 3
+)
+
+const tpl = `{
```

Comment:
I'm confused, isn't this just plain json? What is the benefit of using template.

Reply:
remove done.